README.md

```diff
@@ -1,2 +1,50 @@
-# medium-guidelines
-Guidelines and good practices for Medium publications
+# Medium Guidelines
+
+These guidelines are made for people who love writing amazing stories on Medium. As developers, we often miss the opportunity to get benefits from sharing our experiences, knowledge and thinking. We created this space to share best practices based on our own experiences to help everyone to be able to get the most out of Medium.
```

Comment:
@damienfraud I was thinking about being explicit about that is based on our experiences, but I feel it should stay implicit. For example when you go to (Airbnb/javascript](https://github.com/airbnb/javascript) guideline repository, you know that it's based on their experiences. Being too explicit make people uncomfortable to contribute to it.